solution/ui/regulations/eregs-vite/src/utilities/filters.js

```diff
@@ -0,0 +1,47 @@
+const formatDate = (dateString) => {
+    const date = new Date(dateString);
+    let options = { year: "numeric", timeZone: "UTC" };
+    const raw_date = dateString.split("-");
+    if (raw_date.length > 1) {
+        options.month = "long";
+    }
+    if (raw_date.length > 2) {
+        options.day = "numeric";
+    }
+    const format = new Intl.DateTimeFormat("en-US", options);
+    return format.format(date);
+};
```

Comment:
Template filters are no longer supported in Vue 3, so I'm moving some out into a global filters file and just importing them into components and using them as normal JS functions.